chalk/core.py

```diff
@@ -189,18 +164,6 @@ def beside(self, other: "Diagram") -> "Diagram":
     __or__ = beside
 
     def above(self, other: "Diagram") -> "Diagram":
-        """Given two diagrams ``a`` and ``b``, ``a.above(b)``
-        places ``b`` under ``a``. This moves ``b`` up to
-        touch ``a``.
-
-        💡 ``a.above(b)`` is equivalent to ``a / b``.
-
-        Args:
-            other (Diagram): Another diagram object.
-
-        Returns:
-            Diagram: A diagram object.
-        """
```

Comment:
Same as [this comment](https://github.com/danoneata/chalk/pull/58/files#r893055676).